app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -466,6 +467,18 @@ protected int getSuccessfulSends() {
         return successes;
     }
 
+    protected String getLabelForFormsSent() {

```

Comment:
@amstone326 Ok. I was trying to be more explicit with a numeral value rather than letting user identify the singular 'Form', kind of more on lines of Android Plurals. But yeah can revert it back. Also what should be the case if user cancelled the task before any form was uploaded (in the form processing phase itself) - Right now we say '0 forms sent to the server'. We can change it to 'No forms sent to the server' if that sounds more correct.